app/src/main/java/com/rentminder/Icon&Text.kt

```diff
@@ -35,6 +32,9 @@ fun RentIconText() {
     }
 }
 
+/**
+ * Composable function for the Electric Icon text
+ */
```

Comment:
For KDoc to be valuable, it should add description over and above what can be read by the method name. 

This is redundant with the method name, and thus, isn't adding any value.